src/components/Datatable/Dropdown/Dropdown.tsx

```diff
@@ -49,6 +46,15 @@ export const DropdownLink = styled.button<DropdownLinkProps>`
   }
 `;
 
+const placementAdapter = (input: string): Placements => {
```

Comment:
This is also only internal component so we should rather use original values. I would even go further and completely removed the `placement` prop and staticaly set the BaseDropdown `placement` prop to `bottom-end`. And remove prop set from https://github.com/securityscorecard/design-system/blob/alpha/src/components/_internal/BaseTable/columns/actionsColumn.tsx#L70